features/google/src/model/google-account.js

```diff
@@ -0,0 +1,99 @@
+//TODO: Use DataAccount super class
+export default class GoogleAccount {
+    constructor() {
+        this._importingResults = [];
+        this._importedFileNames = [];
+
+        this._name = "";
+        this._preferredLanguage = [];
+        this._analyses = {};
+        this._reports = {};
+        this._analysesExecutionResults = [];
+        this._placeVisits = [];
+        this._activitySegments = [];
+        this._activities = [];
+        this._pathNames = [];
+    }
+
+    get analyses() {
+        return this._analyses;
+    }
+
+    get reports() {
+        return this._reports;
+    }
+
+    get importedFileNames() {
+        return this._importedFileNames;
+    }
+
+    addImportedFileName(fileName) {
+        this._importedFileNames.push(fileName);
+    }
+
+    get importingResults() {
+        return this._importingResults;
+    }
+
+    set importingResults(importingResults) {
+        this._importingResults = importingResults;
+    }
+
+    get analysesExecutionResults() {
+        return this._analysesExecutionResults;
+    }
+
+    set analysesExecutionResults(analysesExecutionResults) {
+        this._analysesExecutionResults = analysesExecutionResults;
+    }
+
+    get placeVisits() {
+        return this._placeVisits;
+    }
+
+    set placeVisits(placeVisits) {
+        this._placeVisits = placeVisits;
+    }
+
+    get activitySegments() {
+        return this._activitySegments;
+    }
+
+    set activitySegments(activitySegments) {
+        this._activitySegments = activitySegments;
+    }
+
+    get activities() {
+        return this._activities;
+    }
+
+    set activities(activities) {
+        this._activities = activities;
+    }
+
+    get pathNames() {
+        return this._pathNames;
+    }
+
+    set pathNames(pathNames) {
```

Comment:
So many getters and setters... Can't think of an alternative? Is there going to be a mechanism for maintaining internal consistency? Or this is going to be just the deposit of results?

Reply:
Most of them are in the dataAccount class that is being worked on. I didn't want to base this PR on the other one as the other one is already based of another one so I added this temporary solution. The other PR will show up very soon when they are merged https://github.com/polypoly-eu/polyPod/pull/824 and https://github.com/polypoly-eu/polyPod/pull/826. Let's discuss it there